client/logger

```diff
@@ -0,0 +1 @@
+../cloud/logger
```

Comment:
This was attempt to make logger available for both client and cloud. 
Because we wanted to avoid duplication and another "common" category I have created symlink to that module so it will compile and will be available to consume in UI. 
Current ui has lots of console.log and we do not plan to change that but that may be useful for anyone starting to write new UI and using logger in other client side components.